getting_started/introduction/learning_new_features.rst

```diff
@@ -123,11 +123,11 @@ information:
    should have a built-in tool to take screenshots with the :kbd:`PrtSc` (Print
    Screen) key.
 
-   Alternatively, you can use a program like `ShareX <https://getsharex.com/>`_
+   Alternatively, you can use a program like `Snipping Tool <https://support.microsoft.com/en-us/windows/use-snipping-tool-to-capture-screenshots-00246869-1843-655f-f220-97299b865f6b>`_
    on Windows or `FlameShot <https://flameshot.org/>`_ on Linux.
 
 5. Sharing a video of your running game can also be really **useful to
-   troubleshoot your game**. You can use programs like `OBS Studio
+   troubleshoot your game**. On Windows, you can use the built-in `Snipping Tool <https://www.microsoft.com/en-us/windows/learning-center/how-to-record-screen-windows-11>`_. For Windows, OSX, and Linux, you can use programs like `OBS Studio
```

Comment:
```suggestion
   troubleshoot your game**. On Windows, you can use the built-in `Snipping Tool <https://www.microsoft.com/en-us/windows/learning-center/how-to-record-screen-windows-11>`_. For Windows, macOS, and Linux, you can use programs like `OBS Studio
```
OSX is not used since 2018.

Also, macOS has a built-in screen recording tool (https://support.apple.com/en-us/102618) for both screenshots and recordings, as well as most Linux distros, so recommending OBS specifically for macOS/Linux is a bit strange.